schema.cc

```diff
@@ -1113,6 +1114,14 @@ schema_ptr schema_builder::build() {
     return make_lw_shared<schema>(schema(new_raw, _view_info));
 }
 
+void schema_builder::build_properties_derived_from_extensions() {
```

Comment:
This is a 100% CDC-only method. Does it need to be in schema.cc/hh?
Moreover, why do we even need the "_cdc_options" schema member any more, if it directly calculated from the new "schema extension"?

Reply:
Changed _cdc_options field to a cdc_options() method.